runtime/heiko/src/lib.rs

```diff
@@ -257,6 +269,12 @@ impl orml_tokens::Config for Runtime {
     type WeightInfo = ();
     type ExistentialDeposits = ExistentialDeposits;
     type MaxLocks = MaxLocks;
+    type DustRemovalWhitelist = DustRemovalWhitelist;
```

Comment:
Is it a new functionality in `orml_token`, can you explain how it works?

Reply:
yes it's new functionality

Reply:
@MrPai https://github.com/open-web3-stack/open-runtime-module-library/blob/master/tokens/src/lib.rs#L792